airbyte-integrations/connectors/source-harvest/source_harvest/schemas/company.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I think the file name should be `contacts.json`.

Reply:
You're right. My slip up.